docs/uploader/aws-s3-multipart.mdx

```diff
@@ -191,7 +191,7 @@ const uppy = new Uppy()
 :::warning
 
 Until the next major version, not setting this option uses the
-[legacy version of this plugin](./aws-s3). This is a suboptimal experience for
+[legacy version of this plugin](../aws-s3). This is a suboptimal experience for
```

Comment:
This change looks odd, `aws-s3.mdx` is a sibling of `aws-s3-multipart.mdx`, I wonder if there's a way to configure Docosorus to "do the right thing" here. In the mean time, let's add a trailing slash to reflect the runtime behavior.
```suggestion
[legacy version of this plugin](../aws-s3/). This is a suboptimal experience for
```

Reply:
Using `../` is correct in this case. It's because we're moving between directories in the address bar. We start at `/docs/aws-s3-multipart/` and want to go to `/docs/aws-s3/`. To do this, we first go up to the parent directory, which is `docs`, and then into the `aws-s3` directory.

When we used `./`, the browser thought we were trying to find something inside the current directory, `/docs/aws-s3-multipart/`. But what we needed was to go up one level first, then into the correct directory. That's why `../` is the right choice.